native-image-workshop/README.md

```diff
@@ -214,14 +231,14 @@ OK, so now we have added logging, let's see if it works by rebuilding and runnin
 
 ![User Input](./images/userinput.png)
 ```sh
-$ mvn clean package exec:exec -PJAVA
+$ mvn clean package exec:exec
 ```
 
 Great, that works. Now, let's build a native image using the maven profile:
 
 ![User Input](./images/userinput.png)
 ```sh
-$ mvn clean package -PNATIVE_IMAGE
+$ mvn clean package -Pnative
 ```
 
 The run the built image:
```

Comment:
Running this native image fails with an exception:

Reply:
updated - The . Then; native image > executable. But otherwise, kept the flow the same